src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -260,6 +260,10 @@ private void handleNotificationsAndAudit(UserAccountEntity userAccountEntity,
     private void validateUpdateTranscription(TranscriptionEntity transcription,
                                              UpdateTranscription updateTranscription) {
 
+
+        if (getUserAccount().getUserName().equals(transcription.getRequestor())) {
```

Comment:
is it definitely .equals or .equalsIgnoreCase - this tripped me up on a bug recently